charts/hpe-flexvolume-driver/v3.0.0/questions.yml

```diff
@@ -0,0 +1,78 @@
+categories:
```

Comment:
can we please add the following labels to the `questions.yml` for Rancher to use it, thanks
```
rancher_min_version: 2.3.0-rc1
labels:
  io.cattle.role: cluster # options are cluster/project
  io.rancher.certified: partner
```

Reply:
thanks, added it.